src/openrct2/platform/Crash.cpp

```diff
@@ -39,6 +42,31 @@ const wchar_t* _wszCommitSha1Short = WSZ("");
 // OPENRCT2_ARCHITECTURE is required to be defined in version.h
 const wchar_t* _wszArchitecture = WSZ(OPENRCT2_ARCHITECTURE);
 
+static bool UploadMinidump(const wchar_t* dumpPath)
+{
+    std::wstring url(
+        L"https://submit.backtrace.io/openrct2/e9e6d681fafdeac9f6131b4b59a155d54bebad567a8c0380d70643f4414819f5/minidump");
```

Comment:
This could be extracted to a constant at the top of the file. What does the long hex string represent?

Reply:
Access token